p/themes/Alternative-Dark/adark.css

```diff
@@ -211,19 +211,19 @@ a.btn {
 	height: 2.5em;
 	line-height: 2.5em;
 	font-size: 0.9rem;
+	color: #ccc;
 }
 
-.nav-list .item:hover {
-	background: #1f1f1f;
+.nav-list .item a {
+	color: #ccc;
 }
 
-.nav-list .item.active {
-	background: #171717;
-	color: #fff;
+.nav-list .item a:hover{
+	background-color: #1f1f1f;
 }
 
-.nav-list .item.active a {
-	color: #fff;
+.nav-list .item.active a{
```

Comment:
I assume the CI will complain about these but leaving a comment here in case it doesn't.

Reply:
I haven't clean the CI messages right now.

Reply:
fixed